package.json

```diff
@@ -36,12 +36,13 @@
 		"*": "prettier --ignore-unknown --write"
 	},
 	"devDependencies": {
+		"@clack/core": "^0.3.2",
+		"@clack/prompts": "^0.6.3",
```

Comment:
Looks like you'll need to check in the updated `pnpm-lock.yaml` too. Builds are failing because the lockfile hasn't been updated for the new list of dev dependencies.

Reply:
I went ahead and resolved the conflicts in packages.json, and regenerated the pnpm lock file, so everything should be good now. Let me know if it all looks good to you now.